package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@google-cloud/oslogin",
```

Comment:
Should we name this `os-login`? We usually concat separate words with a hyphen, e.g. `video-intelligence`.

Reply:
And just for extra info, the generator did originally spit out `videointelligence`, and we had to change it to add the hyphen.

Reply:
I'm not sure. Since @lukesneeringer created repository as `nodejs-oslogin` (and not `nodejs-os-login`, compare to `nodejs-video-intelligence`), I would probably stick to a version without a hyphen. Just like `bigquery` and `bigtable` :)

Reply:
If we have the time, I would push for another opinion before sticking with it. BigQuery is a single word, as is Bigtable. Since the docs upstream title this "OS Login", I think we have a different situation.